libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -11,7 +11,7 @@
     </div>
     <div>
       <button
-        mat-icon-button
+        mat-icon-button aria-label="Remove Reading"
```

Comment:
"Remove Reading"? I'm not sure this makes sense.

Reply:
aria-label is already there, I have removed it.